### DISCUSSION TITLE
Can you hide items/recipes in REI?

### DISCUSSION THREAD

**Discussion by nvrsbr** [shedaniel/RoughlyEnoughItems#938] [Q & A] [answered]
I need to hide some items in REI, does REI have this function?

**Comment by shedaniel** [comment, marked as answer]
What recipes do you wish to hide? REI automatically hides recipes when all of the items in that recipe is hidden, perhaps that's what you want?

Otherwise, I recommend using KubeJS or CraftTweaker to remove and modify recipes.

**Reply by nvrsbr** [reply]
I have removed some recipes from Mekanism with Craft Tweaker.
They have been disabled but the items still show up in REI.
I would like those items not to show up in REI.

**Reply by shedaniel** [reply]
If you scroll down to the bottom of the REI config menu, you will find a button for Filtering, click that and click Manual Filtering, now you can filter the items there.

**Reply by nvrsbr** [reply]
That is exactly what I needed thank you :)
